.github/workflows/labeler.yml

```
@@ -1,16 +1,28 @@
-  contents: read
-  pull-requests: write
-
-jobs:
-  labeler:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/labeler@v6
-        with:
-          repo-token: "${{ secrets.GITHUB_TOKEN }}"
\ No newline at end of file
+name: "Pull Request Labeler"
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)